sinks/prometheus/prometheus.go

```diff
@@ -0,0 +1,143 @@
+package prometheus
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net"
+	"net/url"
+	"strings"
+	"text/template"
+
+	"github.com/stripe/veneur/samplers"
+	"github.com/stripe/veneur/sinks"
+	"github.com/stripe/veneur/ssf"
+	"github.com/stripe/veneur/trace"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	// Max package size for TCP (and hardcoded for UDP in Statsd Exporter) is
+	// 65535 bytes. Assuming a generous 50 bytes for metric name, 10 bytes for
+	// value, and 200 bytes for tags, we can have ~200 packets per batch.
+	batchSize = 200
```

Comment:
Is there a reason we made this a var and not a const?

Also, consts can be any statically-evaluated expression, so you could actually do `const batchSize = 65535/(200 +  10 + 50)`, which evaluates to 252 via integer division. Though

Reply:
Main reason is for testing purposes -- in one of the tests we change batchSize so we can see the batching logic in action

Reply:
Not sure if having the calculations there make it more understandable (vs the mega comment), since it is extremely hand wavey

Reply:
Ah, okay, that makes sense (re: batching).

I'm fine either way here - the comment is descriptive enough. Just mentioning it as an FYI.